cost-analyzer/templates/cost-analyzer-ingestion-configmap.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if eq (include "aggregator.deployMethod" .) "statefulset" }}
```

Comment:
Do you think it's a requirement we check that Aggregator is being deployed as a StatefulSet? If so, do we need to perform this same check in `_helpers.tpl`?

Reply:
It's not a requirement, just that `statefulset` is the only valid deployment method of a fed primary aggregator, so I thought it would be worth checking. There's not really any risk.

Reply:
I'd opt to remove it! I think >90% of enterprise configurations will use the StatefulSet, but a few don't.

Reply:
Done.